src/templates/DatasetListSubmenu/DatasetListSubmenu.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useContext, useEffect, useState } from 'react';
```

Comment:
This is pretty nitpicky - but typically "templates" are reserved for full page layouts that are then filled with smaller components.

The header menu to me seems like it's a better fit for a component than a template.

That said, our templates and components conventions are definitely in need of an organizational overhaul 😓

Reply:
Not nitpicky at all thanks! I'm glad to understand the conventions. I'll re-review and move the components where they belong.